client/wordpress-com.js

```diff
@@ -193,6 +193,16 @@ sections.push( {
 	secondary: true
 } );
 
+if ( config.isEnabled( 'account-recovery' ) ) {
+	sections.push( {
+		name: 'account-recovery',
+		paths: [ '/account-recovery' ],
+		module: 'account-recovery',
+		secondary: false,
+		enableLoggedOut: true,
```

Comment:
@gwwar / @aduth Is it enough to just not include `/** @ssr-ready **/` so that this doesn't get rendered server side? I think since this does a redirect if the user is logged in and probably will do other stuff based on api calls then its probably not a good candidate.